### PR TITLE
Tables' links and icons improved

### PR DESCRIPTION
**Pull request by FatimaMubeenSh**

### CODE REVIEW

File: app/views/companies/display_surveys.html.erb [hunk 1]
```diff
@@ -1,24 +1,29 @@
...
-      <button class="primary-btn btn-lg dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown">
-        <%= t('public_surveys.filter_label') %>
-      </button>
-      <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-        <a href="#_" class="dropdown-item" data-toggle="modal" data-target="#name_modal"><%= t('public_surveys.name_label') %></a>
-        <a href="#_" class="dropdown-item" data-toggle="modal" data-target="#survey_type_modal"><%= t('public_surveys.type_label') %></a>
-        <a href="#_" class="dropdown-item" data-toggle="modal" data-target="#expired_before_modal"><%= t('public_surveys.expired_before_label') %></a>
-        <a href="#_" class="dropdown-item" data-toggle="modal" data-target="#created_on_modal"><%= t('public_surveys.created_on_label') %></a>
-        <a href="#_" class="dropdown-item" data-toggle="modal" data-target="#created_by_modal"><%= t('public_surveys.created_by_label') %></a>
+  <% if can? :create, Survey %>
```
Comment (zohaibbutt13) [inline app/views/companies/display_surveys.html.erb]: if user cannot create survey then he cannot filter as well ?

File: app/views/companies/display_surveys.html.erb [hunk 2]
```diff
@@ -1,24 +1,29 @@
...
+          <a href="#_" class="dropdown-item" data-toggle="modal" data-target="#created_by_modal"><%= t('public_surveys.created_by_label') %></a>
+        </div>
+      </div>
+      <div class="col-2 mb-2">
+        <td><%= link_to t('surveys.new_survey_link'), new_survey_path, class: 'primary-btn' %></td>
       </div>
     </div>
+  <% end %>
+
+  <div class="row d-flex justify-content-start">
```
Comment (zohaibbutt13) [inline app/views/companies/display_surveys.html.erb]: do we need this div ?

File: app/views/companies/display_surveys.html.erb (hunk 1 above)
Comment (FatimaMubeenSh) [reply app/views/companies/display_surveys.html.erb]: changed

File: app/views/companies/display_surveys.html.erb (hunk 2 above)
Comment (FatimaMubeenSh) [reply app/views/companies/display_surveys.html.erb]: not needed, removed

File: app/views/companies/_list_surveys.html.erb [hunk 3]
```diff
@@ -13,26 +12,33 @@
...
+          <% end %>
           <% if (can? :edit, survey) && (survey.expiry > DateTime.now) %>
-            <%= link_to t('links.edit_link'), edit_survey_path(survey) %>
+            <%= link_to edit_survey_path(survey) do %>
+              <i class="fa fa-pencil mr-2"></i>
+            <% end %>
           <% end %>
           <% if can? :destroy, survey %>
-            <%= link_to t('links.delete_link'), survey_path(survey), method: :delete, data: { confirm: t('links.confirmation_message') } %>
+            <%= link_to survey_path(survey), method: :delete, data: { confirm: t('links.confirmation_message') } do %>
```
Comment (aliahmad7) [inline app/views/companies/_list_surveys.html.erb:36]: sweet alert

File: app/views/companies/display_surveys.html.erb [hunk 4]
```diff
@@ -19,7 +16,13 @@
...
       </div>
     </div>
+    <div class="col-2 mb-2">
+      <% if can? :create, Survey %>
+        <td><%= link_to t('surveys.new_survey_link'), new_survey_path, class: 'primary-btn' %></td>
+      <% end %>
+    </div>
   </div>
+
   <% if @surveys.count > 0 %>
```
Comment (aliahmad7) [inline app/views/companies/display_surveys.html.erb]: count length size

File: app/views/companies/_list_surveys.html.erb (hunk 3 above)
Comment (FatimaMubeenSh) [reply app/views/companies/_list_surveys.html.erb:36]: added

File: app/views/companies/display_surveys.html.erb (hunk 4 above)
Comment (FatimaMubeenSh) [reply app/views/companies/display_surveys.html.erb]: resolved

File: app/views/companies/_list_surveys.html.erb (hunk 3 above)
Comment (FatimaMubeenSh) [reply app/views/companies/_list_surveys.html.erb:36]: used

**Pull request merged**